entries/lane/src/main/java/graders/AbstractGrader.java

```diff
@@ -0,0 +1,17 @@
+package graders;
```

Comment:
Instead of clone and own, can you not just import the entries:followsclosely module and use the classes?